src/main/kotlin/carracing/RandomIntCarRacingManager.kt

```diff
@@ -0,0 +1,19 @@
+package carracing
+
+class RandomIntCarRacingManager(
+    carList: CarList,
+    randomIntRacingMovementRole: RandomIntRacingMovementRole
+) : CarRacingManager<Int>(carList, randomIntRacingMovementRole) {
+    companion object {
+        fun fromInputDto(
+            inputDto: InputDto,
+            randomIntRacingMovementRole: RandomIntRacingMovementRole
+        ): RandomIntCarRacingManager {
+            var inputCars = emptyList<Car>()
+            repeat(inputDto.carCount) {
+                inputCars = inputCars + Car()
+            }
```

Comment:
`List`를 이용하여 아래와 같이 작성이 가능해보여요!

```suggestion
           val cars = List(inputDto.carCount) { Car() }
```